eslint.config.js

```diff
@@ -6,8 +6,18 @@ import html from 'eslint-plugin-html';
 import noOnlyTests from 'eslint-plugin-no-only-tests';
 import simpleImportSort from 'eslint-plugin-simple-import-sort';
 import globals from 'globals';
+import licenseHeader from './custom-rules/eslint/license-header.js';
 
 export default [
+  {
+    plugins: {
+      'custom-rules': {
```

Comment:
This can be moved to the section below where other plugins (`es-x` etc) are declared.

Reply:
Done.